include/material/mohr_coulomb.tcc

```diff
@@ -277,36 +312,41 @@ Eigen::Matrix<double, 6, 1> mpm::MohrCoulomb<Tdim>::compute_stress(
   // Friction and dilation in radians
   const double phi_max = friction_angle_ * PI / 180.;
   const double psi_max = dilation_angle_ * PI / 180.;
-  const double c_max   = cohesion_;
+  const double c_max = cohesion_;
   const double phi_min = residual_friction_angle_ * PI / 180.;
   const double psi_min = residual_dilation_angle_ * PI / 180.;
-  const double c_min   = residual_cohesion_;
+  const double c_min = residual_cohesion_;
   // Current MC parameters using a linear softening rule
-  Eigen::Matrix<double, 6, 1> PDS = ptr-> plastic_deviatoric_strain ();
-  const double epds = (2. / 3.) * sqrt(3. / 2. * (PDS(0) * PDS(0) + PDS(1) * PDS(1) + PDS(2) * PDS(2))
-                       + 3. * (PDS(3) * PDS(3) + PDS(4) * PDS(4) + PDS(5) * PDS(5)));
+  Eigen::Matrix<double, 6, 1> PDS = ptr->plastic_deviatoric_strain();
+  const double epds =
+      (2. / 3.) *
+      sqrt(3. / 2. * (PDS(0) * PDS(0) + PDS(1) * PDS(1) + PDS(2) * PDS(2)) +
+           3. * (PDS(3) * PDS(3) + PDS(4) * PDS(4) + PDS(5) * PDS(5)));
   double j2, j3, rho, theta;
   Vector6d trial_stress = stress + (this->de_ * dstrain);
   this->compute_rho_theta(stress, j2, j3, rho, theta);
   double epsilon = (1. / sqrt(3.)) * (stress(0) + stress(1) + stress(2));
-  Eigen::Matrix<double,2,1> yield_function , yield_function_trial;
-  phi_ = phi_max; psi_ = psi_max; c_ = c_max;
+  Eigen::Matrix<double, 2, 1> yield_function, yield_function_trial;
+  phi_ = phi_max;
```

Comment:
Let's remove assigning `phi_`, `psi_` & `c_` to its maximum values here. Insted place this initialization begining of the analysis, i.e. in the `MohrCoulomb`.